### ISSUE TITLE
Sublime text 4 support 

### ISSUE THREAD

**Issue by iamgm** [rctay/sublime-text-buildview#37]
I've installed sublime text 4 and it seems that it doesn't support buildview. Moreover buildview menu item in Preferences -> Package settings disappeared. I know there is an alternative of using terminus instead. But I found it not so convenient.  Do you have any plans on updating the package?

**Comment by YoSTEALTH** [comment]
I would also like to see support for sublime text 4. This is a great little feature.

**Comment by YoSTEALTH** [comment]
You can get this project working for sublime text 4 by replacing
`if sublime.version().startswith('3'):` with `if sublime.version().startswith(('3', '4')):`
in files `pipe_views.py` and `commands.py`

Also adding `"show_panel_on_build": false,` into `Preferences -> Setting` if you want to hide default build panel

**Comment by rctay** [comment]
Thanks everyone, let me sort out some Package Control stuff and a new release for Sublime Text 4 should be coming soon.

**Comment by YoSTEALTH** [comment]
The new changes seem to open new build panel on every build vs using the previous one.

could you change:
``` python
import re
import sublime, sublime_plugin

if re.match('3|4', sublime.version()):
```
with:
``` python
import sublime
import sublime_plugin
from distutils.version import LooseVersion

if LooseVersion(sublime.version()) >= '3':
```
for both `pipe_views.py` and `commands.py` files.

`LooseVersion` is proper way to handle version comparison. 

Thanks @rctay

**Comment by rctay** [comment]
Hi @YoSTEALTH, I just tried this using the plugin as published right now, and I couldn't reproduce what you said (new output build panels for every new build).

My setup:
- Sublime Text 4107
- install buildview from Package Control (ensured latest, ie. ran Package Control -> Upgrade Packages from Command Palette, no updates)
Steps:
- Quit Sublime Text 4107
- Start Sublime Text 4107
- Open file with code (this is the dummy code I usually use):
  ```python
import datetime
import time

for i in range(15):
	print(datetime.datetime.now())
	print("foo %d" % i)
	time.sleep(0.1)

  ```
- Press Ctrl-B
  - Saw new tab, "Build output" created with lines starting "2021-05-29 08:45:03..."
- Upon build completed, Press Ctrl-B
  - Saw output streamed in the previously created "Build output" tab with lines starting "2021-05-29 08:47:22..."


Could you copy/replace the steps above with the steps you used to run into the problem? That would be helpful in reproducing the problem.

**Comment by iamgm** [comment]
Thanks for replies.

I also have the same issue with new version.
> The new changes seem to open new build panel on every build vs using the previous one. 

and for me
there is no build output in a build tab at all.
I don't insist there is a problem in `buildview`. Most likely it is in my setup. 

Also I discovered there are 2 similar packages in package control now. One of them is `buildview`, and second one is `sublime-text-buildview`. Tried both.  Both gave same results. 

Actually I more or less set up terminus + origami pair that day when the issue was opened.  But anyway using 2 packages for solving one problem looks like weird workaround.
